docs/doc/13-sql-reference/30-table-engines/00-fuse.md

```diff
@@ -32,6 +32,8 @@ The `CLUSTER BY` parameter specifies the sorting method for data that consists o
 
 Fuse engine support following common case-insensitive options:
 
+- `bloom_index_columns = '<column> [, <column> ...]'`, specify the bloom index columns. The data type of `column` could be Map, Number, String, Date, or Timestamp. Bloom index default to be created on all supported columns.
```

Comment:
We need to introcude a special case: 
```
alter table t set options(bloom_index_columns='');
```
will disable all the bloom filter indexing.